keyboards/uranuma/keymaps/default/keymap.c

```diff
@@ -0,0 +1,80 @@
+#include QMK_KEYBOARD_H
+
+enum layer_number {
+  _BASE,
+  _LOWER,
+  _RAISE,
+  _ADJUST,
+};
+
+enum custom_keycodes {
+  BASE = SAFE_RANGE,
+  KANJI,
+  LOWER,
+  RAISE,
+};
```

Comment:
`KANJI` is the only keycode you're using here, and it doesn't have any functionality written.

```suggestion
enum custom_keycodes {
  KANJI = SAFE_RANGE,
};
```